src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/hooks/useEventList.js

```diff
@@ -23,19 +23,21 @@ const basedFieldTypes = [
     { type: dataElementTypes.DATE },
     { type: dataElementTypes.UNKNOWN, resolveValue: convertCommentForView },
 ];
-const baseColumnHeaders = [
+const getBaseColumnHeaders = props => [
     { header: i18n.t('Status'), sortDirection: SORT_DIRECTION.DEFAULT, isPredefined: true },
-    { header: i18n.t('Report date'), sortDirection: SORT_DIRECTION.DEFAULT, isPredefined: true },
-    { header: i18n.t('Registering unit'), sortDirection: SORT_DIRECTION.DEFAULT, isPredefined: true },
     {
-        header: i18n.t('Due date'), sortDirection: SORT_DIRECTION.DEFAULT, isPredefined: true,
+        header: props.formFoundation.getLabel('occurredAt'),
+        sortDirection: SORT_DIRECTION.DEFAULT,
+        isPredefined: true,
     },
+    { header: i18n.t('Registering unit'), sortDirection: SORT_DIRECTION.DEFAULT, isPredefined: true },
+    { header: i18n.t('Due date'), sortDirection: SORT_DIRECTION.DEFAULT, isPredefined: true },
```

Comment:
Hey,

The fix looks good for 'Report date' label. 
How about the other hardcoded name 'Due date' mentioned in the ticket? I believe something like this 
`{ header: props.formFoundation.getLabel('scheduledAt'), ... },` can fix it. WDYT?

Thank you!

Reply:
Thanks @simonadomnisoru I updated the scheduleAt as well